nifi-nar-bundles/nifi-extension-utils/nifi-database-utils/src/test/java/org/apache/nifi/util/db/TestJdbcCommon.java

```diff
@@ -465,7 +474,12 @@ private void testConvertToAvroStreamForBigDecimal(BigDecimal bigDecimal, int dbP
             assertEquals("decimal", logicalType.getName());
             LogicalTypes.Decimal decimalType = (LogicalTypes.Decimal) logicalType;
             assertEquals(expectedPrecision, decimalType.getPrecision());
-            assertEquals(expectedScale, decimalType.getScale());
+            // If scale > precision then precision will be set to the scale value
+            if (expectedScale > expectedPrecision) {
+                assertEquals(expectedScale, decimalType.getScale());
+            } else {
+                assertEquals(expectedScale, decimalType.getScale());
```

Comment:
This assertion is the same as the one used when `expectedScale` is greater than `expectedPrecision`, should this be different?

Reply:
Oops that's leftover from when I tried setting scale to precision instead of the other way around. Will condense into a single check, good catch